kostal/README.md

```diff
@@ -1,97 +1,201 @@
 
-### Example
+If your communication board has a firmwareversion 5 (datastructure=html),
+the following values are not displayed in the html-status page and also not
+available for the plugin.
 
-Example configuration which shows the current status and the current, total and
-daily power. Additionally it shows the volts and watts for the phases.
+dctot_w, dc1_w, dc2_w, dc3_w,
+actot_cos, actot_limitation, ac1_a, ac2_a, ac3_a,
+operationtime_h
+
+Hint:
+Item names have changed from the previous version of the plugin, so that both
+types of communication can be configured the same way.
 
 <pre>
 # items/my.conf
 [solar]
     [[status]]
+        name = Wechselrichter Status
         type = str
-        kostal = status
-    [[current]]
+        kostal = operation_status
+    [[dcpower]]
+        name = Gleichspannungsleitung gesamt
+        type = num
+        kostal = dctot_w
+    [[dc1_v]]
+        name = DC-Strang 1 Spannung
+        type = num
+        kostal = dc1_v
+    [[dc1_a]]
+        name = DC-Strang 1 Strom
         type = num
-        kostal = power_current
-    [[total]]
+        kostal = dc1_a
+    [[dc1_w]]
+        name = DC-Strang 1 Leistung
         type = num
-        kostal = power_total
-    [[day]]
+        kostal = dc1_w
+    [[dc2_v]]
+        name = DC-Strang 2 Spannung
         type = num
-        kostal = power_day
-    [[l1v]]
+        kostal = dc2_v
+    [[dc2_a]]
+        name = DC-Strang 2 Strom
         type = num
-        kostal = l1_watt
-    [[l1w]]
+        kostal = dc2_a
+    [[dc2_w]]
+        name = DC-Strang 2 Leistung
         type = num
-        kostal = l1_watt
-    [[l2v]]
+        kostal = dc2_w
+    [[actot_w]]
+        name = Wechselspannungsleistung gesamt
         type = num
-        kostal = l2_watt
-    [[l2w]]
+        kostal = actot_w
+    [[actot_cos]]
+        name = Cos phi
         type = num
-        kostal = l2_watt
-    [[l3v]]
+        kostal = actot_cos
+    [[actot_limitation]]
+        name = Abregelungsfaktor
         type = num
-        kostal = l3_watt
-    [[l3w]]
+        kostal = actot_limitation
+    [[ac1_v]]
+        name = Phase 1 Spannung
         type = num
-        kostal = l3_watt
+        kostal = ac1_v
+    [[ac1_a]]
+        name = Phase 1 Strom
+        type = num
+        kostal = ac1_a
+    [[ac1_w]]
+        name = Phase 1 Leistung
+        type = num
+        kostal = ac1_w
+    [[ac2_v]]
+        name = Phase 2 Spannung
+        type = num
+        kostal = ac2_v
+    [[ac2_a]]
+        name = Phase 2 Strom
+        type = num
+        kostal = ac2_a
+    [[ac2_w]]
+        name = Phase 2 Leistung
+        type = num
+        kostal = ac2_w
+    [[ac3_v]]
+        name = Phase 3 Spannung
+        type = num
+        kostal = ac3_v
+    [[ac3_a]]
+        name = Phase 3 Strom
+        type = num
+        kostal = ac3_a
+    [[ac3_w]]
+        name = Phase 3 Leistung
+        type = num
+        kostal = ac3_w
```

Comment:
I don't see such a value for the HTML version, but it could easily calculated by the other two values (`ac3_a` and `ac3_v`). Do you think it make sense to support as most as possible values for both types of converters?